test/Conversion/LinalgToTpp/linalg-to-tpp.mlir

```diff
@@ -14,31 +14,6 @@ func.func @brgemm_lowering(%arg0: memref<3x5x4xf32>, %arg1: memref<3x4x5xf32>,
 
 // -----
 
-#map = affine_map<(d0, d1, d2) -> (d0, d1, d2)>
-
-// CHECK-LABEL: func.func @relu(
-func.func @relu(%arg3: memref<64x32x32xf32>) -> memref<64x32x32xf32> {
```

Comment:
Do we have those tests on 2D tensors? If not, we should jus change them and make sure we still match the 2D tiles as a test.

Reply:
yes, multiple.